web-client/public/js/container.js

```diff
@@ -4,23 +4,42 @@ $(function () {
   var enableScroll = $("#enableScroll");
   var pageWidth = $(window).width();
 
+  var WIDTH_OFFSET = 250;
+  var HEIGHT_OFFSET = 53;
+
+  var windowWidth = $(window).width() - WIDTH_OFFSET;
+  var windowHeight = $(window).height() - HEIGHT_OFFSET;
+
   if (pageWidth < 1500) {
       $('#boundBoxS').prop('checked', true);
-      $(".containerDefault").attr("class", "grnsight-container containerS");
+      container.addClass("containerS");
   } else if (pageWidth > 1500 && pageWidth < 2200) {
       $('#boundBoxM').prop('checked', true);
-      $(".containerDefault").attr("class", "grnsight-container containerM");
+      container.addClass("containerM");
   } else {
       $('#boundBoxL').prop('checked', true);
-      $(".containerDefault").attr("class", "grnsight-container containerL");
+      container.addClass("containerL");
   }
 
   $(".boundBoxSize").on("click", function () {
     var currentValue = $(this).val();
     var grnsightContainerClass = "grnsight-container " + currentValue;
     if (!container.hasClass(currentValue)) {
       container.attr("class", grnsightContainerClass);
+      if (currentValue === "containerFit") {
+          container.css({width: windowWidth, height: windowHeight});
+      } else {
+          container.css({width: "", height: ""});
+      }
```

Comment:
NBD on this one, but according to the structure, I would prefer a conditional expression:

    container.css(currentValue === "containerFit" ?
        {width: windowWidth, height: windowHeight} :
        {width: "", height: ""}
    );

...to me, this emphasizes better that this code block is setting the CSS and is choosing between two options.